lib/gui/components/safe-webview/safe-webview.js

```diff
@@ -84,6 +84,14 @@ class SafeWebview extends react.PureComponent {
       this.refs.webview.addEventListener('new-window', this.constructor.newWindow);
       this.refs.webview.addEventListener('console-message', this.constructor.consoleMessage);
 
+      if (this.props.onStateChange) {
+        this.props.onStateChange((event, toState, toParams, fromState) => {
+          if (fromState.name === 'success' && this.state.shouldLoad) {
+            this.refs.webview.src = this.props.src;
```

Comment:
@alexandrosm Can we tell from the analytics how many people leave Etcher open for an extended period of time? (my gut instinct would be not many people at all)  How important is it that users always see the 'latest' banner, rather than the banner that they downloaded when they started Etcher?